azure-pipelines.yml

```diff
@@ -23,6 +23,12 @@ steps:
     echo '##vso[task.prependpath]$(GOROOT)/bin'
   displayName: 'Set up the Go workspace'
 
+- script: |
+    go get github.com/gobuffalo/packr/packr
+    go get github.com/jmhodges/jsonpp
+    export PATH=${GOBIN}:${PATH}
```

Comment:
GOBIN is already magically on the path, thanks to an odd looking line farther up in the file. `echo '##vso[task.prependpath]$(GOBIN)'`

Reply:
Ah ok 👍